spec/namespaces/indices.yaml

```diff
@@ -2972,7 +2972,7 @@ components:
       description: |-
         Name of the data stream, which must meet the following criteria:
         Lowercase only;
-        Cannot include `\`, `/`, `*`, `?`, `"`, `<`, `>`, `|`, `,`, `#`, `:`, or a space character;
```

Comment:
why is this needed?

Reply:
getting lint error with python generator: invalid escape sequence ' \ `'

Reply:
And also an error while generating python client docs.
expected doc:
`Cannot include ‘\’, ‘/’`
obtained:
`Cannot include ‘’, ‘/’`

Reply:
Try replacing `\` with `\\` in the descriptions

Reply:
@nhtruong, just to give you some background: Here are the two things
- [Generating APIs](https://github.com/opensearch-project/opensearch-py/blob/main/DEVELOPER_GUIDE.md#client-code-generator) from the spec
       Here, param descriptions will be generated as it is. 

- [Building documentation](https://github.com/opensearch-project/opensearch-py/blob/main/DEVELOPER_GUIDE.md#documentation) using Sphinx.

- Currently, 
 Generated API:`Cannot include ‘\’, ‘/’` 
 Sphinx documentation:`Cannot include ‘’, ‘/’` 

- When replaced \ with \\,
 Generated API:`Cannot include ‘\\’, ‘/’`          **not as expected**
 Sphinx documentation:`Cannot include ‘\’, ‘/’` 


Do you think I should create a workaround in Python? I'm ready to do it if necessary, but I'm also worried that other clients might face the same problem.

Reply:
That seems right @saimedhi. Having a `\` is valid in a spec, but may need to be converted in other formats (python). This is similar to HTML which encodes characters like `<` to `&lt;`.